README.md

```diff
@@ -19,7 +21,34 @@ python3 -m uvicorn fastapi_app:app --reload --port=8000
 
 ```bash
 cd src/frontend
+npm install
 npm run dev
 ```
 
-1. Open the browser at `http://localhost:5173/` and you will see the frontend.
\ No newline at end of file
+1. Open the browser at `http://localhost:5173/` and you will see the frontend.
+
+## Setup permissions for Azure OpenAI endpoint
+
+If the permisions have not yet been setup, you can configure them using the following script:
+
+```
+export AZURE_RESOURCE_GROUP=REPLACE_ME
+export AZURE_PRINCIPAL_ID=$(az ad signed-in-user show --output tsv --query "id")
+export AZURE_SUBSCRIPTION_ID=$(az account show --query "name" --out tsv)
+
+roles=(
+    "5e0bd9bd-7b93-4f28-af87-19fc36ad61bd" # Cognitive Services OpenAI User
+    "2a2b9908-6ea1-4ae2-8e65-a410df84e7d1" # Storage Blob Data Reader
```

Comment:
Hm, it shouldnt need the bottom 4 roles here?

Reply:
Not sure, I copied the roles from that other repo
https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/scripts/roles.sh

Reply:
I'll try without those roles and see if it works

Reply:
Ah okay, we shouldnt need those here, as the only service we need locally is OpenAI. The other one needs those for local citation viewing and AI Search usage.

Reply:
So I tried to remove the other permissions but gave up as there is a huge lag for permissions to be effective, which makes narrowing down quite tricky :/